### ISSUE TITLE
Error: Cannot download this app 240 Bad request

### ISSUE THREAD

**Issue by rodrie03** [Lekensteyn/apk-downloader#15]
I can't download games with instruction provided, I trying download free applications using this program.  My device is compatible and the app is been distributed in my country, I just downloaded one to my android phone using the Install buttom and it works but can't directly using the apk-downloader

**Comment by Lekensteyn** [comment]
240 is not a valid response code. Did you mean 400? Is the device ID the same as the one where you installed the app?

**Comment by rodrie03** [comment]
Yes, it appears on screen 240. No, isn't the same device, the device id is from a Samsung Android phone, just want to download the games apps to my computer using apk and then to a nabi 2
